velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -372,4 +372,18 @@ struct AddMonthsFunction {
     result = daysSinceEpoch;
   }
 };
+
+template <typename T>
+struct QuarterFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE int32_t getQuarter(const std::tm& time) {
```

Comment:
It may be better to move this to `TimeUtils.h` for possible reusing at other places.

Reply:
@PHILO-HE Updated.